WalletWasabi.Tests/UnitTests/Tor/TorSettingsTests.cs

```diff
@@ -13,20 +14,18 @@ public class TorSettingsTests
 		[Fact]
 		public void GetCmdArgumentsTest()
 		{
-			var settings = new TorSettings("temp", "temp/Tor.log", "temp/TorDaemons");
+			var settings = new TorSettings(Path.Combine("temp", "tempdata"), Path.Combine("temp", "Tor.log"), "tempdist");
```

Comment:
This line is very confusing. What about:

```suggestion
			var settings = new TorSettings(Path.Combine("temp", "tempDataDir"), Path.Combine("temp", "tempDataDir", "Tor.log"), Path.Combine("temp", "tempDistribution"));
```

It's not great, but it looks better to me. Those non-camel case paths are hard to read. (Why Bitcoin actually uses that is beyond me.)

Reply:
ACK